### PR TITLE
Use Chrome instead of PhantomJS to run tests

### PR DESCRIPTION
**Pull request by Gaurav0**
Breaking.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,6 @@ cache:
 before_install:
   - curl -o- -L https://yarnpkg.com/install.sh | bash
   - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn global add phantomjs-prebuilt
-  - phantomjs --version
```
Comment (Turbo87) [inline .travis.yml]: I think this is missing the `addon: Chrome` thing

File: .travis.yml [hunk 2]
```diff
@@ -4,14 +4,18 @@ language: node_js
 node_js:
   - "6"
 
+sudo: false
```
Comment (Turbo87) [inline .travis.yml]: this is already set on line 1

File: .travis.yml [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
 node_js:
   - "6"
 
+addons:
+  chrome: stable
+
 cache:
   yarn: true
 
+before_script:
```
Comment (rwjblue) [inline .travis.yml]: Hmm, this shouldn't be needed...

Comment (Gaurav0) [reply .travis.yml]: You are correct. It should be using the --no-sandbox and --headless mode. It isn't.

Comment (rwjblue) [reply .travis.yml]: Possibly hardcoded resolutions forcing an older version of testem that isn't aware of the new syntax (nested `ci` property under `Chrome` section)/

Comment (Gaurav0) [reply .travis.yml]: I already tried forcing the latest testem. Let me try using an older syntax in testem.js

Comment (Gaurav0) [reply .travis.yml]: @rwjblue Yay that worked!

File: .travis.yml [hunk 4]
```diff
@@ -1,17 +1,19 @@
-sudo: false
+sudo: required
```
Comment (rwjblue) [inline .travis.yml]: Why do we need `sudo`?

Comment (Gaurav0) [reply .travis.yml]: Ah, let me try removing that. We needed it for the sandbox stuff.

Comment (Gaurav0) [reply .travis.yml]: @rwjblue removed.

**Comment by Turbo87** [comment]
@Gaurav0 can you squash the commits so that the history is a little easier to read?

**Comment by Gaurav0** [comment]
Sure thing.

**Pull request merged**